fudge-ratio/src/main/webapp/landing.css

```diff
@@ -1,160 +1,136 @@
 @import url('https://fonts.googleapis.com/css2?family=Do+Hyeon&family=Open+Sans:wght@400;700&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Bad+Script&family=Open+Sans:wght@400;700&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&family=Poiret+One&display=swap');
+
 :root {
   --background-color: rgb(229, 231, 225);
-  --accent-color:rgba(45, 93, 122, 0.712);
+  --accent-color: rgba(45, 93, 122, 0.75);
+  --font-color: rgb(37, 51, 51);
   --base: min(3.5vw, 1.7rem);
+  --font-base: calc(1.2vw + 0.8rem);
```

Comment:
In this way you have a responsive font (using ``vw``) when you change the width of the screen and using ``rem``